test/integration/038_caching_tests/test_caching.py

```diff
@@ -1,67 +0,0 @@
-from test.integration.base import DBTIntegrationTest, use_profile
-from dbt.adapters.factory import FACTORY
-
-class TestBaseCaching(DBTIntegrationTest):
-    @property
-    def schema(self):
-        return "caching_038"
-
-    @property
-    def project_config(self):
-        return {
-            'config-version': 2,
-            'quoting': {
-                'identifier': False,
-                'schema': False,
-            }
-        }
-
-    def run_and_get_adapter(self):
-        # we want to inspect the adapter that dbt used for the run, which is
-        # not self.adapter. You can't do this until after you've run dbt once.
-        self.run_dbt(['run'])
-        return FACTORY.adapters[self.adapter_type]
-
-    def cache_run(self):
-        adapter = self.run_and_get_adapter()
-        self.assertEqual(len(adapter.cache.relations), 1)
-        relation = next(iter(adapter.cache.relations.values()))
-        self.assertEqual(relation.inner.schema, self.unique_schema())
-        self.assertEqual(relation.schema, self.unique_schema().lower())
-
-        self.run_dbt(['run'])
```

Comment:
If I followed this logic right: This second run was not actually re-returning the adapter, so `relation` and `second_relation` were being pulled from the same `adapter.cache.relations`, and we were just asserting equality between pointers to the same object in memory.